exasol/slc/internal/tasks/upload/upload_container_tasks_creator.py

```diff
@@ -7,24 +7,29 @@
     RequiredTaskInfo,
 )
 
+import exasol.slc.internal.tasks.upload.upload_containers
 from exasol.slc.internal.tasks.upload.upload_container_task import UploadContainerTask
-from exasol.slc.internal.tasks.upload.upload_containers_parameter import (
-    UploadContainersParameter,
-)
 
 
 class UploadContainerTasksCreator:
 
-    def __init__(self, task: UploadContainersParameter):
+    def __init__(
+        self,
+        task: "exasol.slc.internal.tasks.upload.upload_containers.UploadFlavorContainers",
```

Comment:
Why is this necessary?

Reply:
Need to use this annotation because otherwise would have a cyclic dependency
because UploadFlavorContainers also imports UploadContainerTasksCreator